examples/dash/dash-connect-pgsql/README.md

```diff
@@ -0,0 +1,28 @@
+# Dash App Connected to PostgreSQL Database
+
+Interactive Dash Application, connected to PostgreSQL database.
+
+![](app.png)
+
+To use the app, fill in the environment variables (database parameters) in Dockerfile with the parameters from your PostgreSQL database. 
+
+```DockerFile
+FROM python:3.11-slim-bookworm
+
+WORKDIR /app
+
+COPY requirements.txt /app/
+
+RUN python3 -m pip install -r requirements.txt --no-cache-dir
+
+COPY . /app
+
+ENV PGHOST='YOUR_HOST'
+ENV PGDATABASE='test'
+ENV PGUSER='test_owner'
+ENV PGPASSWORD='your_password'
+
+RUN python3 upload.py
```

Comment:
[removed Dockerfile](https://github.com/ploomber/doc/pull/259/commits/88873fd30aec1e0b4f5c2ffd26088c8c5b5d0510#diff-947610ca10aacde7e2a54afa9c3fbd37fea9055230c4a6222dc39decedc110afL8-L27) from README since it's redundant.